Cargo.toml

```diff
@@ -33,6 +33,7 @@ futures = "0.1.14"
 gfx = "0.16"
 num_cpus = "1.2"
 genmesh = "0.4"
+image = "0.15"
```

Comment:
we already have _imagefmt_. Is this one better?
If that's the case then remove _imagefmt_ from dependencies.